test/hw_tests/hwtest_scan_compare.cpp

```diff
@@ -17,7 +17,6 @@
 #include <gtest/gtest.h>
 
 #include <boost/filesystem.hpp>
-#include <boost/function.hpp>
 #include <boost/bind.hpp>
```

Comment:
```suggestion
```
Where is bind used in `scan_compare.cpp`? Are there more unused boost modules?

Reply:
Not used. Removed.